std/conv.d

```diff
@@ -5687,10 +5688,6 @@ pure nothrow @safe /* @nogc */ unittest
 
 @system unittest
 {
-    int var = 6;
```

Comment:
Why are these lines deleted?

Reply:
Because I moved the content of a `version (unittest)` inside a `unittest {}`, so I had to move those lines too. They are now at line 5009.